editorconfig.go

```diff
@@ -16,6 +16,8 @@ const (
 	ConfigNameDefault = ".editorconfig"
 	// UnsetValue is the value that unsets a preexisting variable.
 	UnsetValue = "unset"
+	// OffValue is the value that disables max_length (non-standard).
+	OffValue = "off"
```

Comment:
Couldn't we use `unset` here too? What's the difference between toggling this off and unsetting it?

Reply:
`unset` means restore default value, this is more like a `false`.. but yeah... 

https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#max_line_length

Reply:
I agree that `off` is a bad naming... https://github.com/go-gitea/gitea/issues/20694